src/components/Buttons.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+interface Props {
+  tabButtons: Tab[];
+  changeTab: (id: string) => void;
+}
+
+export const Buttons: React.FC<Props> = (props) => {
+  const { tabButtons, changeTab } = props;
+
+  return (
+    <div>
+      {tabButtons.map(tab => (
+        <button
+          type="button"
+          key={tab.id}
+          onClick={() => changeTab(tab.id)}
```

Comment:
you still call callback every time on clicking the tab (even if this tab is already opened)
![image](https://user-images.githubusercontent.com/64437769/134162676-947af27f-cc01-47e1-a548-54a8b4f89ee0.png)